pop11_grammar_ebnf.txt

```diff
@@ -25,8 +29,8 @@ Definition ::=
 	'define' Word ('(' Varslist? ')' | Varslist)? ('->' Varslist)* ';' Sequence 'enddefine'
 
 Assignment ::=
-	Expression ('->' | '->>') (Variable | FunctionApplication)
-	| Expression '-->' Expression
+	NonEmptyExpression ('->' | '->>') (Variable | FunctionApplication)
+	| NonEmptyExpression '-->' NonEmptyExpression

```

Comment:
Incorrect.  Setpop
 : vars x, y;
 : vars p = [?x ?y];
 : [99 88];
 : --> p;
 : 
 : x =>
 ** [] 99
 : y =>
 ** 88
 : Pop11 allows for the LHS to be empty.
```

```